docs/membership.md

```diff
@@ -0,0 +1,165 @@
+reopository that they are active in.
+
+### Approver
+
+Code approvers are able to both review and approve code contributions.  While
+code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, subtle performance and correctness issues, alignment with upstream 
+features and plans, and in general are the drivers of the project.  
+
+**Defined by:** Inclusion in the Approver team in the Universal Blue GitHub organization
+
+#### Requirements
+
+- Member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase 
+- Demonstrate sound technical judgement
+- Responsible for project quality control via code reviews
+  - With a focus on long term sustainability to the project
+- Expected to be responsive to review requests as per [community expectations]
+- Mentor new contributors and members
+- Deep understanding of our working relationship with Fedora and other upstreams
+  - Is invested in keeping track of upstream issues that affect our repositories
+  - Is comfortable reporting and testing issues upstream
+  
+#### Responsibilities and privileges
+
+- Approver status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgement
+  - Is actively monitoring upstream projects and participating accordingly as good open source citizens.
+  - This may include rejecting ideas in order to maintain project scope
+- Responsible for project quality control via code reviews
+- Expected to be responsive to review requests as per [community expectations]
+- Mentor new contributors and reviewers
+
+## Inactive members
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Universal Blue GitHub organizations comes with
+an elevated set of permissions. These capabilities should not be used by those
+that are not familiar with the current state of the project.
+
+Therefore members with an extended period away from the project with no activity
+will be removed from the Universal Blue GitHub Organizations and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+### How inactivity is measured
+
+Inactive members are defined as members of one of the Universal Blue Organizations
+with **no** contributions across any organization over the past 6 months. This is
+measured by GitHub activity.
+
+After an extended period away from the project with no activity
+those members would need to re-familiarize themselves with the current state
+before being able to contribute effectively.
+
+
+## Owner
```

Comment:
Wait, so is Owner a community status, or just a GH status?  This section doesn't make that clear.

Reply:
Made that more explicit, thanks for the review so far Josh, it means a lot!